locale/en/manageiq.po

```diff
@@ -30830,6 +30893,10 @@ msgstr ""
 msgid "Enter tag category to map to"
 msgstr ""
 
+#: ../app/javascript/components/request-workflow-status/data.js
+msgid "Entered Time"
```

Comment:
I still think this wording is strange - I'm wondering if "Started Time" or "Started At" is better. cc @agrare Thoughts?

Reply:
`EnteredTime` is what aws step-functions has as the time the state started (and `FinishedTime` is the bookend).  We can call it whatever we want I suppose but this is likely the reason.

They are a little inconsistent in that the StateMachine uses `StartTime` / `EndTime` and the States use `EnteredTime`/`FinishedTime`

We could standardize on `Started` / `Finished` for the workflow and the states

Reply:
Yeah, Started/Finished feels like what we would use elsewhere in the app as well, so I like that one.